### PR TITLE
Fix client in VLM JS inference snippet

### PR DESCRIPTION
**Pull request by Wauplin**
Small typo. In JS, the library should be `huggingface.js` not `huggingface_hub`.

### CODE REVIEW

**Review by mishig25 — APPROVED** [review]

**Review by julien-c — APPROVED** [review]

File: packages/tasks/src/snippets/js.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const snippetTextGeneration = (
 		if (streaming) {
 			return [
 				{
-					client: "huggingface_hub",
+					client: "huggingface.js",
```
[the remark this answers is not in the document]
Comment (Wauplin) [reply packages/tasks/src/snippets/js.ts:60]: I'll let @coyotte508 decide what the best name is between `huggingface.js` and `@huggingface/inference`

[the remark this answers is not in the document]
Comment (coyotte508) [reply packages/tasks/src/snippets/js.ts:60]: where is the `huggingface.js` shown?

[the remark this answers is not in the document]
Comment (Wauplin) [reply packages/tasks/src/snippets/js.ts:60]: here: https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct?inference_api=true
Go to "JavaScript". At the moment it's `huggingface_hub`. In this PR I changed it to `huggingface.js` but happy to change it to `@huggingface/inference` if it's better.

[the remark this answers is not in the document]
Comment (julien-c) [reply packages/tasks/src/snippets/js.ts:60]: i think `huggingface.js` is good

**Pull request merged**